CoreImportExportPlugins/test/unit/src/au/gov/asd/tac/constellation/plugins/importexport/ConfigurationPaneNGTest.java

```diff
@@ -41,7 +34,7 @@
  * @author altair1673
  */
 public class ConfigurationPaneNGTest {
-
+    private static final Logger LOGGER = Logger.getLogger("ConfigurationPaneNGTest");
```

Comment:
Are you able to replace the text in the logger instantiation with `ConfigurationPaneNGTest.class.getName()`